.changeset/honest-trainers-remain.md

```diff
@@ -0,0 +1,6 @@
+---
+"@ckb-lumos/common-scripts": minor
+"@ckb-lumos/config-manager": minor
+---
+
+feat: omnilock bitcoin auth message is changed to `CKB (Bitcoin Layer) transaction: 0x`
```

Comment:
Is this a breaking change?

Reply:
the behavior here is more like a fixture since it doesn't work now if we don't update the magic PREFIX to be the same with the latest testnet Omnilock

Reply:
Got it